### PR TITLE
Pcx 9497

### PR DESCRIPTION
**Pull request by jason-cf**

### CODE REVIEW

File: content/security-center/infrastructure/index.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ weight: 5
 
 # Review the infrastructure
 
+{{<render file="_temporary-disable.md" productFolder="security-center" withParameters="Infrastructure">}}
```
Comment (marciocloudflare) [inline content/security-center/infrastructure/index.md]:
```suggestion
{{<render file="_temporary-disable.md" withParameters="Infrastructure">}}
```

File: content/security-center/security-insights/_index.md [hunk 2]
```diff
@@ -6,6 +6,8 @@ weight: 4
 
 # Security Insights
 
+{{<render file="_temporary-disable.md" productFolder="security-center" withParameters="Security Insights">}}
```
Comment (marciocloudflare) [inline content/security-center/security-insights/_index.md]:
```suggestion
{{<render file="_temporary-disable.md" withParameters="Security Insights">}} 
```

File: content/security-center/infrastructure/index.md (hunk 1 above)
Comment (marciocloudflare) [reply content/security-center/infrastructure/index.md]: The `productFolder=""` argument is only needed outside the folder where you have the partial file

File: content/security-center/security-insights/how-it-works.md [hunk 3]
```diff
@@ -6,6 +6,8 @@ weight: 1
 
 # How it works
 
+{{<render file="_temporary-disable.md" productFolder="security-center" withParameters="Security Insights">}}
```
Comment (marciocloudflare) [inline content/security-center/security-insights/how-it-works.md]:
```suggestion
{{<render file="_temporary-disable.md" withParameters="Security Insights">}}
```

File: content/security-center/security-insights/review-insights.md [hunk 4]
```diff
@@ -6,6 +6,8 @@ weight: 2
 
 # Review Security Insights
 
+{{<render file="_temporary-disable.md" productFolder="security-center" withParameters="Security Insights">}}
```
Comment (marciocloudflare) [inline content/security-center/security-insights/review-insights.md]:
```suggestion
{{<render file="_temporary-disable.md" withParameters="Security Insights">}}
```

**Review by marciocloudflare — APPROVED** [review]

**Comment by XoL1507** [comment]
i don’t know what happen


Sent from Yahoo Mail for iPhone


On Thursday, January 18, 2024, 7:14 PM, jason-cf ***@***.***> wrote:


You can view, comment on, or merge this pull request online at:

  https://github.com/cloudflare/cloudflare-docs/pull/12588

Commit Summary
   
   - bc1a302 first draft of banner
   - e95565c adding banners indicating that certain tabs are not available at this time

File Changes 
 
(5 files)
   
   -  A content/security-center/_partials/_temporary-disable.md (11) 
   -  M content/security-center/infrastructure/index.md (2) 
   -  M content/security-center/security-insights/_index.md (2) 
   -  M content/security-center/security-insights/how-it-works.md (2) 
   -  M content/security-center/security-insights/review-insights.md (2) 

Patch Links:
   
   - https://github.com/cloudflare/cloudflare-docs/pull/12588.patch
   - https://github.com/cloudflare/cloudflare-docs/pull/12588.diff

**Pull request merged**